api/app/applicationModel/api.py

```diff
@@ -79,18 +88,101 @@ def get(self):
             for s in form.sections:
                 s.questions = []
                 for q in questions:
-                    if(q.section_id == s.id):
+                    if (q.section_id == s.id):
                         s.questions.append(q)
 
-            if (form): 
+            if (form):
                 return marshal(form, self.form_fields)
             else:
-                LOGGER.warn("Event not found for event_id: {}".format(args['event_id'])) 
+                LOGGER.warn("Event not found for event_id: {}".format(args['event_id']))
                 return EVENT_NOT_FOUND
 
         except SQLAlchemyError as e:
             LOGGER.error("Database error encountered: {}".format(e))
             return DB_NOT_AVAILABLE
-        except: 
+        except:
             LOGGER.error("Encountered unknown error: {}".format(traceback.format_exc()))
             return DB_NOT_AVAILABLE
+
+    @auth_required
+    def post(self):
+        args = self.req_parser.parse_args()
+        event_id = args['event_id']
+
+        event = db.session.query(Event).get(event_id)
+        if not event:
+            return EVENT_NOT_FOUND
+
+        user_id = g.current_user["id"]
+        current_user = user_repository.get_by_id(user_id)
+        if not current_user.is_admin:
+            return FORBIDDEN
+
+        app_form = app_repository.get_by_event_id(event_id)
+        if app_form:
+            return APPLICATION_FORM_EXISTS
+        else:
+            is_open = args['is_open']
+            nominations = args['nominations']
+
+            app_form = ApplicationForm(
+                event_id,
+                is_open,
+                nominations
+            )
+            db.session.add()
+
+        section_args = args['section']
+        section = Section(section_args)
```

Comment:
You need to create the section objects inside the loop - so loop over section_args and each item in section_args will be the dictionary of fields for that particular section